src/themes/default/pages/Product.vue

```diff
@@ -293,6 +293,7 @@ export default {
         this.product.configurable_options.forEach(configurableOption => {
           const type = configurableOption.attribute_code
           const filterVariants = configurableOption.values.map(({value_index, label}) => {
+            label = label ? label : this.options[type].find(config => config.id === value_index).label
```

Comment:
There should be a check if the ` this.options[type].find(config => config.id === value_index) !== null`

Reply:
I'm also not sure how it could happen that the `label` is not set for the `configurable_options` item - can you elaborate on when did you have such a problem?

Reply:
> I'm also not sure how it could happen that the `label` is not set for the `configurable_options` item - can you elaborate on when did you have such a problem?

@pkarw, Thank you for quick reply.

I can't get label in `this.product.configurable_options` object. i'm getting only `value_index` from there.
I've checked elastic search data also, At there only `value_index` is stored.
Check it out here:
![2019_07_10_Selection_003](https://user-images.githubusercontent.com/38535982/60945294-cdd70f00-a308-11e9-8acb-4bce44ecb720.png)

Reply:
Thanks for reply - are you using `mage2vuestorefront` as an indexer? It's setting the label: https://github.com/DivanteLtd/mage2vuestorefront/blob/c71788e1c918225685344657f9d6ca0b2417d801/src/adapters/magento/product.js#L486

Reply:
Yes, I'm using `mage2vuestorefront`.